alicloud/resource_alicloud_cs_kubernetes.go

```diff
@@ -57,24 +68,55 @@ func resourceAlicloudCSKubernetes() *schema.Resource {
 			},
 			"master_instance_type": &schema.Schema{
 				Type:         schema.TypeString,
-				Required:     true,
+				Optional:     true,
 				ValidateFunc: validateInstanceType,
+				Deprecated:   "please use 'master_instance_types' instead",
+			},
+			"master_instance_types": &schema.Schema{
+				Type:     schema.TypeList,
+				Optional: true,
+				ForceNew: true,
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
```

Comment:
If the original field is "required", the instead field also should be "required".